jcp/parser.js

```diff
@@ -43,19 +43,19 @@ module.exports = new Parser(function analyseEC(parsedUrl, ec) {
       result.unitid = itemid[1];
     }
 
-    if (match[1] === 'JCP' || match[1] === 'jcp') {
+    if (match[1].toLowerCase() === 'jcp') {
       result.doi = jcp_doi_prefix + itemid[1];
-    } else if (match[1] === 'PCC' || match[1] === 'pcc') {
+    } else if (match[1].toLowerCase() === 'pcc') {
       result.doi = pcc_doi_prefix + itemid[1];
     }
   } else if ((match = /^\/([a-zA-Z]{3})\/article\/(Pages|pages)\/\d\d\d\d\/[a-z0-9]*\/(.*)\.aspx$/i.exec(path)) !== null) {
     // http://www.psychiatrist.com:80/JCP/article/Pages/2018/v79n02/17m11587.aspx
     result.rtype    = 'ARTICLE';
     result.mime     = 'HTML';
     result.unitid   = match[3];
-    if (match[1] === 'jcp' || match[1] === 'JCP') {
+    if (match[1].toLowerCase() === 'JCP') {
```

Comment:
bad condition, the toLowerCase puts in lower case content of match[1] so you have to put JCP in lower case.